gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileSystemProviderTest.java

```diff
@@ -518,7 +509,8 @@ public void testIsDirectory_trailingSlash_alwaysTrue() throws Exception {
   @Test
   public void testIsDirectory_trailingSlash_pseudoDirectoriesDisabled_false() throws Exception {
     try (CloudStorageFileSystem fs = forBucket("doodle", usePseudoDirectories(false))) {
-      assertThat(Files.isDirectory(fs.getPath("fundir/"))).isFalse();
+      CloudStoragePath path = fs.getPath("fundir/");
+      assertThat(Files.isDirectory(path)).isFalse();
```

Comment:
go/unit-test-practices recommends inlining values that are only referenced once.